Source/GameEngine/Game.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GameEngine
+{
+    public class Game
+    {
+        public Game()
+        {
+            Console.WriteLine("New game of Ludo");
```

Comment:
Woho!! Finally! This is what I've been waiting for my whole life. A brand new game of Ludo :)